libvast/vast/system/node_state.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef VAST_SYSTEM_NODE_STATE_HPP
+#define VAST_SYSTEM_NODE_STATE_HPP
+
+#include "vast/expression.hpp"
+#include "vast/filesystem.hpp"
+
+#include "vast/system/tracker.hpp"
+
+namespace vast {
+namespace system {
+
+/// A container for VAST components.
+struct node_state {
+  path dir;
+  tracker_type tracker;
+  std::unordered_map<std::string, int> labels;
+  std::string name = "node";
+  std::unordered_map<expression, caf::actor> continuous_queries;
```

Comment:
Don't think this is currently in use. Wanted to add booking to allow the NODE to pass existing queries to IMPORTERs that are spawned later on.

Would it make sense to add this data to the tracker? Doesn't it already track running EXPORTERs?